airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/display_report.py

```diff
@@ -187,11 +234,15 @@ def report_init_endpoint(self, record_type: str) -> str:
     metrics_type_to_id_map = METRICS_TYPE_TO_ID_MAP
 
     def _get_init_report_body(self, report_date: str, record_type: str, profile):
+        metrics_list = self.metrics_map[record_type]
         if record_type == RecordType.ASINS and profile.accountInfo.type == "vendor":
             return None
+        elif record_type == RecordType.PRODUCTADS and profile.accountInfo.type != "seller":
+            # Remove SKU from metrics since it is only available for seller accounts in Product Ad report
+            metrics_list = [m for m in metrics_list if m != "sku"]
```

Comment:
am i understand this correctly, by removing this from the list if the customer has the seller account this property will not be added to the report?

Reply:
If the profile is not that of a seller, we remove the `sku` from the list. As a result, the API request will not include the `sku` metric. If we attempt to pass this for a vendor profile, for instance, it will result in an error.